demisto_sdk/commands/common/content/objects/pack_objects/pack.py

```diff
@@ -24,13 +27,18 @@
     SecretIgnore, Widget)
 from demisto_sdk.commands.common.content.objects_factory import \
     path_to_pack_object
+from demisto_sdk.commands.test_content import tools
 from wcmatch.pathlib import Path
 
+TURN_VERIFICATION_ERROR_MSG = "Can not turn on the pack verification, In the server - go to Settings -> troubleshooting\
```

Comment:
i delete the verification key flag in server when try to turn on the verification (delete the key == turn on)
and the reason why to delete it is because i don't want to leave not required key (key = true) as in most of cases the key was not exist before the command run
so in case of error the message ask the user to delete it manually